packages/sanity/src/core/studio/screens/schemaErrors/SchemaProblemGroups.tsx

```diff
@@ -57,7 +59,8 @@ export function SchemaProblemGroups(props: {problemGroups: SchemaValidationProbl
                 <Text size={1} weight="semibold">
                   {schemaType ? (
                     <>
-                      {capitalize(schemaType.type)} type "{schemaType.name}"
+                      {capitalize(schemaType.type)} {t('navbar.configuration.type-label')} "
```

Comment:
**suggestion:** this could be a good candidate for the `<Trans>` component? Currently this renders `Document` `type` `"book"` which may need to be re-ordered in specific locales?

Reply:
Good point! Replacing it!